app/views/schools/_details_form.html.erb

```diff
@@ -17,6 +17,8 @@
     <%= f.input :activation_date, as: :tempus_dominus_date, default_date: nil, hint: t('schools.school_details.activation_date_hint') %>
     <%= f.input :enable_targets_feature %>
     <%= f.input :public %>
+    <%= f.label :funder %>
+    <%= f.select :funder_id, options_for_select(Funder.all.pluck(:name, :id), school.funder_id), { include_blank: true }, { class: 'form-control' } %>
```

Comment:
I did suggest adding it to the configuration form rather than the detailed form, but I'd forgotten we had some admin specific things here. At some point I'd like to have all the admin options on the configuration form `/schools/:id/configuration/edit` rather than on this form as it ensure that the admin options are all in one place.

Reply:
:+1: I've added a funder field to the configuration form too